pyproject.toml

```diff
@@ -327,3 +327,13 @@ markers = [
     "containers: tests which need to spawn containers",
     "web: tests which need to access the web"
     ]
+
+[tool.codespell]
+ignore-words = "docs/codespell.dic"
+exclude-file = "docs/codespell.ignore"
+# Synchronize with .pre-commit-config.yaml
+skip = [
+    "tests/execute/weird/data/weird.txt",
+    "tests/lint/plan/data/invalid_attr.fmf",
+    "tests/lint/plan/test.sh",
+]
```

Comment:
```suggestion
skip = "tests/execute/weird/data/weird.txt,tests/lint/plan/data/invalid_attr.fmf,tests/lint/plan/test.sh"
```

Reply:
It accepts comma-separated string :shrug: 
With it, you can get rid of the explicit exclude in pre-commit config.

Reply:
Wow, weird that it behaves like that, thanks for that find